product_equivalent_category/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds the concept of equivalent category. A product can
+belong to a certain equivalent category and all products in that
+category will be considered equivalent.
```

Comment:
Please, could you explain what it mean?
What is the different in Odoo if 2 products belong to the same equivalent category?

Reply:
I extended the description file :+1:  Thanks for the feedback